src/modules/task/internal/store/task-repository.ts

```diff
@@ -0,0 +1,24 @@
+import mongoose, { CallbackError, Connection } from 'mongoose';
+import ConfigService from '../../../config/config-service';
+import { TaskDB, taskDBSchema } from './task-db';
+
+export default class TaskRepository {
+  public static task: mongoose.Model<TaskDB>;
```

Comment:
rename to `taskDb`

Reply:
on using the name ```taskDb``` linting is throwing error to use name in PascalCase, so I will be using ```TaskDb``